packages/printable-itinerary/src/accessibility-annotation.js

```diff
@@ -0,0 +1,43 @@
+import coreUtils from "@opentripplanner/core-utils";
+import React from "react";
+import PropTypes from "prop-types";
+
+import { AccessibilityRating } from "@opentripplanner/itinerary-body";
```

Comment:
This goes below the core-utils import.

Reply:
Fixed in 9b052113f918c4d8c638f99fb623ad71b8bba15d